src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/payments.html.twig

```diff
@@ -38,6 +38,21 @@
           <strong>{{ orderForViewing.payments.paidAmount }}</strong>
           {{ 'paid instead of'|trans({}, 'Admin.Orderscustomers.Notification') }}
           <strong>{{ orderForViewing.payments.amountToPay }}</strong>
+
+          {% if orderForViewing.linkedOrders.linkedOrders is not empty %}
+            {% set firstLinkedOrder = orderForViewing.linkedOrders.linkedOrders|first %}
```

Comment:
`firstLinkedOrder` is not used

Reply:
Indeed, it was a bad copy-paste of legacy
Fixed in https://github.com/PrestaShop/PrestaShop/pull/19704/commits/6cc29a04eae57adf48dc288aa3385fdab0ec3fff